src/main/java/com/commercetools/sync/commons/helpers/CustomReferenceResolver.java

```diff
@@ -107,10 +107,11 @@ protected CustomReferenceResolver(@Nonnull final S options, @Nonnull final TypeS
         try {
             final String customTypeKey = getKeyFromResourceIdentifier(custom.getType());
             return typeService.fetchCachedTypeId(customTypeKey);
-        } catch (ReferenceResolutionException exception) {
+        } catch (ReferenceResolutionException referenceResolutionException) {
```

Comment:
What is the point of your change in here, it does not seem necessary to me. You are not improving anything in here, the handler was also simple and easy to read before.

Reply:
Would base on what implemented in master branch, so I would rollback this change.

Reply:
As suggested, change here has been rolled back.